README.md

```diff
@@ -90,3 +90,8 @@ You can configure the mapping (Auth0 data -> Mautic User data) in the configurat
 </table>
 
 Use dot syntax to access arrays (i.e. `user_metadata.login_name`).
+
+## Update from Mautic 2.x.x to Mautic 3.x.x
+When Updating the Plugin, you have to make sure to change the Callback-URL(s) in Auth0. 
```

Comment:
You can simplify this lines by just writing:
When updating the plugin, please make sure to change the callback URL from `../s/sso_login/Auth0Auth` to `../s/sso_login/Auth0` in your Auth0 application settings.